src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGeneratorOptions.cs

```diff
@@ -102,5 +108,28 @@ private string DefaultSortKeySelector(ApiDescription apiDescription)
         {
             return TagsSelector(apiDescription).First();
         }
+
```

Comment:
I would prefer to see this mapping of ASP.NET schemes to Open API schemes in the `SwaggerGenerator` rather than a config helper. Can you elaborate on the usecase for exposing this as an option at all? 

Also, I generally try to avoid functions that mutate parameters (see line 121). I think it would be better to have this method accept the ASP.NET schemes only and return the corresponding OpenAPI schemes. So, the option is just "bring your own" mapping of ASP.NET to Open API schemes. But again, I would avoid exposing this option unless there's a strong need vs just adding schemes via the `SecuritySchemes` property that's already on this class.